generators/client/templates/angular/src/main/webapp/app/_polyfills.ts

```diff
@@ -16,8 +16,77 @@
  See the License for the specific language governing permissions and
  limitations under the License.
 -%>
-/* tslint:disable */
-import 'reflect-metadata/Reflect';
-import 'zone.js/dist/zone';
+/**
+ * This file includes polyfills needed by Angular and is loaded before the app.
+ * You can add your own extra polyfills to this file.
+ *
+ * This file is divided into 2 sections:
+ *   1. Browser polyfills. These are applied before loading ZoneJS and are sorted by browsers.
+ *   2. Application imports. Files imported after ZoneJS that should be loaded before your main
+ *      file.
+ *
+ * The current setup is for so-called "evergreen" browsers; the last versions of browsers that
+ * automatically update themselves. This includes Safari >= 10, Chrome >= 55 (including Opera),
+ * Edge >= 13 on the desktop, and iOS 10 and Chrome on mobile.
+ *
+ * Learn more in https://angular.io/docs/ts/latest/guide/browser-support.html
+ */
+
+/***************************************************************************************************
+ * BROWSER POLYFILLS
+ */
+
+/** IE9, IE10 and IE11 requires all of the following polyfills. **/
+import 'core-js/es6/symbol';
```

Comment:
I have a problem with this. This forces all these additional JS on every browser and increases bundle size just to support IE, not everyone might support IE
We officially do not support IE9, 10 so no need of those polyfills by default anyway.

As a compromise what I can accept is loading polyfills required by IE11 alone and have the remaining commented out with clear instruction to enable if need to support older IE versions.

Reply:
We also don't need to support older version than IE11. However, following the comments of the Angular CLI template I was not able to find out which of them are not necessary in this case. I will investigate further.

Reply:
According to Angular Docs (https://angular.io/guide/browser-support) IE11 requires "ES6" polyfills My PR follows the Angular CLI example which does not include the whole core-js package but picks some polyfills. It looks to me like all those would be necessary for IE11. Do you see any indication that some might not be necessary?

Reply:
take a look at this https://polyfill.io/v2/docs/features/

Reply:
Thanks. Looking at the information there, I think it would be possible to remove ``core-js/es6/date`` and ``core-js/es6/regexp``. The other polyfills seem to be required even for IE11. The page also shows that the difference of the required polyfills between IE10 and IE11 is not more than 400 bytes (before compression).

I would suggest to keep the polyfills as suggested by Angular CLI to try not to diverge too much from the standard.

If you don't agree, I would suggest the following patch to this PR:

```
...
- import 'core-js/es6/date';
+ // import 'core-js/es6/date'; // Already supported by IE11
  import 'core-js/es6/array';
- import 'core-js/es6/regexp';
+ // import 'core-js/es6/regexp'; // Already supported by IE11
...
```